astrowidgets/core.py

```diff
@@ -112,6 +115,12 @@ def __init__(self, logger=None, image_width=500, image_height=500,
         # Marker
         self.marker = {'type': 'circle', 'color': 'cyan', 'radius': 20}
         self._marktag = 'marktag'
```

Comment:
`self._marktag` is essentially useless now and hence should be removed accordingly from this entire module.